src/components/CompanyEvents.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <h1>Events</h1>
+    <EventsTable :events="events"></EventsTable>
+  </div>
+</template>
+
+<script>
+import AV from "leancloud-storage";
+import EventsTable from "@/components/EventsTable.vue";
+export default {
+  name: "CompanyEvents",
+  data() {
+    return {
+      companyClients: [],
+      events: []
+    };
+  },
+  components: {
+    EventsTable
+  },
+  created() {
+    const vm = this;
+    const clientsQuery = new AV.Query("Client");
+    clientsQuery.equalTo(
+      "company",
+      AV.Object.createWithoutData("Company", vm.$route.params.id)
+    );
+    clientsQuery
+      .find()
+      .then(function(clients) {
+        vm.companyClients = clients;
+        for (const client in vm.companyClients) {
+          console.log(typeof clients);
+          console.log(typeof client);
+          const eventsQuery = new AV.Query("Event");
+          eventsQuery.equalTo("client", client);
+          eventsQuery.equalTo("done", false);
+          eventsQuery
+            .descending("createdAt")
+            .limit(1000)
+            .find()
+            .then(function(events) {
+              for (const event in events) {
+                vm.events.push(event);
+              }
+            })
+            .catch(error => {
+              alert(error);
+            });
+        }
+      })
+      .catch(error => {
+        alert(error);
+      });
```

Comment:
There's a better way to do this using `matchesQuery`, which only needs to make one single API call for the entire purpose. See https://docs.leancloud.app/leanstorage_guide-js.html#hash-1803281728

Reply:
Oh yeah, shoot, this shouldn't be in this pull request, working on company wide events at the moment...